src/main/java/com/kodilla/ecommercee/controller/cart/CartController.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.ecommercee.controller.cart;
+
+import com.kodilla.ecommercee.domain.Product.ProductDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/cart")
+public class CartController {
+
+    @PostMapping("/createCart/{userID}")
+    public void createCart(@PathVariable Long userID) throws NotFoundException {
+
+    }
+
+    @GetMapping("/{cartID}/getProducts")
+    public List<ProductDto> getProducts(@PathVariable Long cartID) throws NotFoundException {
+        List<ProductDto> products = new ArrayList<>();
+        products.add(new ProductDto(1L, "item1", new BigDecimal(5)));
+        products.add(new ProductDto(2L, "item1", new BigDecimal(8)));
+        return products;
+    }
+
+    @PutMapping("/{cartID}/addProducts/{productID}")
+    public void addProduct(@PathVariable Long cartID, @PathVariable Long productID) throws NotFoundException{
```

Comment:
Tu powinieneś zwracać koszyk chociaż z Description oraz cenami produktów w nim zawartych jako np. Stringi to wymagało by zadeklarowania w Map<String,String> productDetails albo dodać listę Stringów jako ciągły tekst descripion + price, ale to może rodzić problem w dalszym operowaniu tymi danymi.